misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickAltStepScreen.jsx

```diff
@@ -0,0 +1,178 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import counterpart from 'counterpart';
+
+import { postStepUnPicked } from '../../../api/picking';
+import { pushHeaderEntry } from '../../../actions/HeaderActions';
+import { updateAltPickingStepQty } from '../../../actions/PickingActions';
+import ButtonWithIndicator from '../../../components/ButtonWithIndicator';
+import * as CompleteStatus from '../../../constants/CompleteStatus';
+import ScreenToaster from '../../../components/ScreenToaster';
+import { toastError } from '../../../utils/toast';
+
+class PickAltStepScreen extends Component {
```

Comment:
IMHO this shall be the same as a regular picking step screen.
The only diff is that u have to pass the `alternativeId` around.

Reply:
Done